lib/routing.py

```diff
@@ -24,9 +24,9 @@
 except ImportError:
     from urllib.parse import urlsplit, parse_qs
 try:
-    from urllib import urlencode
+    from urllib import urlencode, quote as q, unquote as uq
```

Comment:
I personally prefer the original names as it improves code readability.
Only when there are namespace problems or for backward compatibility renaming is warranted.

For example, **q** is a well-known debugging tool: https://pypi.org/project/q/

Reply:
Yeah, I guess. It might just be my preference, but I hate long lines with repeated function names. By your standards, would you prefer this?
```
quoted_string = double_quote(s)

def double_quote(s):
    return quote(quote(s))
```

Reply:
I haven't even determined when or why this double quoting is needed.